sites/feedback/ajax/report-problem-en.html

```diff
@@ -46,53 +46,21 @@
 						</div>
 						<div class="checkbox">
 							<label for="problem5">
-								<input name="problem5" id="problem5" type="checkbox" value="Yes" >Login error when trying to access an account (e.g. My Service Canada Account)
+								<input name="problem5" id="problem5" type="checkbox" value="Yes" >Login error when trying to access an account
 							</label>
 							<input name="problem5" type="hidden" value="">
 						</div>
-						<ul>
-							<li class="checkbox">
-								<label for="problem6a">
-									<input name="problem6" id="problem6a" type="checkbox" value="Yes" >GC Key access
-								</label>
-								<input name="problem6" type="hidden" value="">
-							</li>
-							<li class="checkbox">
-								<label for="problem7">
-									<input name="problem7" id="problem7" type="checkbox" value="Yes" >SecureKey Concierge (Banking Credential) access
-								</label>
-								<input name="problem7" type="hidden" value="">
-							</li>
-							<li class="checkbox">
-								<label for="problem8">
-									<input name="problem8" id="problem8" type="checkbox" value="Yes" >Personal Access Code (PAC) problems or EI Access Code (AC) problems
-								</label>
-								<input name="problem8" type="hidden" value="">
-							</li>
-							<li class="checkbox">
-								<label for="problem9">
-									<input name="problem9" id="problem9" type="checkbox" value="Yes" >Social Insurance Number (SIN) validation problems
-								</label>
-								<input name="problem9" type="hidden" value="">
-							</li>
-							<li class="checkbox">
-								<label for="problem10">
-									<input name="problem10" id="problem10" type="checkbox" value="Yes" >Other login error not in this list
-								</label>
-								<input name="problem10" type="hidden" value="">
-							</li>
-						</ul>
 						<div class="checkbox">
-							<label for="problem11">
-								<input name="problem11" id="problem11" type="checkbox" value="Yes" >I can't find what I'm looking for
+							<label for="problem6">
+								<input name="problem6" id="problem6" type="checkbox" value="Yes" >I can't find what I'm looking for
 							</label>
-							<input name="problem11" type="hidden" value="">
+							<input name="problem6" type="hidden" value="">
 						</div>
 						<div class="checkbox">
-							<label for="problem12">
-								<input name="problem12" id="problem12" type="checkbox" value="Yes" >Other issue not in this list
+							<label for="problem7">
+								<input name="problem7" id="problem7" type="checkbox" value="Yes" >Other issue not in this list
 							</label>
-							<input name="problem12" type="hidden" value="">
+							<input name="problem7" type="hidden" value="">
 						</div>
```

Comment:
The input name should not have been renamed because it will cause a breaking change with the software that are capturing those data.